.env.example

```diff
@@ -0,0 +1,26 @@
+MODE=prod
+
+MAILMAN_HOST=smtp.gmail.com
+MAILMAN_PORT=587
+MAILMAN_EMAIL=example@gmail.com
+MAILMAN_PASSWORD=---
+
+EXCHANGE_CURRENCY_API_KEY=???
+
+API_ADDRESS=api
+API_PORT=8080
+
+CURRENCY_SERVICE_ADDRESS=currency-service
+CURRENCY_SERVICE_PORT=8090
+CURRENCY_SERVICE_SERVER="${CURRENCY_SERVICE_ADDRESS}:${CURRENCY_SERVICE_PORT}"
```

Comment:
CURRENCY_SERVICE_ADDRESS, CURRENCY_SERVICE_PORT is enough information to get client right url

Reply:
This relates to other *_SERVER env variables